recipes/bamutil/meta.yaml

```diff
@@ -8,14 +8,17 @@ source:
   sha256: "f5ec8d5e98a3797742106c3413a4ab1622d8787e38b29b3df4cddb59d77efda5"
 
 build:
-  number: 1
+  number: 2
 
 requirements:
   build:
-    - gcc # [linux]
```

Comment:
Please also add
```yaml
    # curl/openssl are used to download/verify libStatGen
    - curl
    - openssl
```

Reply:
Done! Could it be that curl is part of the default set of dependencies? I ran it through circleci locally and it worked

Reply:
`curl` is already available in the base system, hence it worked fine. If we want to be "good citizens", we are a bit more explicit/robust here.